tests/starttests_psql.sh

```diff
@@ -0,0 +1,58 @@
+#!/bin/bash
+echo "Starting adaguc-server functional tests"
+pushd `dirname $0` > /dev/null
+SCRIPTPATH=`pwd`
+popd > /dev/null
+unset ADAGUC_CONFIG
+unset QUERY_STRING
+export ADAGUC_PATH=${SCRIPTPATH}/../
+export ADAGUC_PATH=`readlink -f ${ADAGUC_PATH}`/
+export PYTHONPATH=${ADAGUC_PATH}/python/lib/
+echo "ADAGUC-Server path is [$ADAGUC_PATH]"
+export ADAGUC_TMP=${ADAGUC_PATH}/tests/tmp/
+export ADAGUC_LOGFILE=${ADAGUC_PATH}/tests/log/
+rm -rf $ADAGUC_LOGFILE && mkdir -p $ADAGUC_LOGFILE
+export ADAGUC_LOGFILE=${ADAGUC_PATH}/tests/log/adaguc-server.log
+export ADAGUC_FONT="${ADAGUC_PATH}/data/fonts/FreeSans.ttf"
+export ADAGUC_ONLINERESOURCE=""
+export ADAGUC_ENABLELOGBUFFER=FALSE
+export ADAGUC_DATASET_DIR=${ADAGUC_PATH}/data/config/datasets/
+export ADAGUC_DATA_DIR=${ADAGUC_PATH}/data/datasets/
+export ADAGUC_AUTOWMS_DIR=${ADAGUC_PATH}/data/datasets/
+ulimit -c unlimited
+
+# Rewrite the xml configuration files used by tests.
+# Unfortunately macos does not use GNU sed, so the sed command differs slightly (-i '' vs -i)
+project_root_dir=`dirname -- "$( readlink -f -- "$0"; )"`
+project_root_dir+="/../data/config"
+if [ -z "${project_root_dir}" ]; then
+    echo "Could not determine root dir of project!"
+    exit 1
+fi
+
+if [[ "$OSTYPE" == "darwin"* ]]; then
+    find "${project_root_dir}" -type f -name '*.xml' | xargs sed -i '' 's;<DataBase dbtype="sqlite" parameters="{ADAGUC_TMP}/adaguc.autoresource.db"/>;<DataBase parameters="{ADAGUC_DB}"/>;g'
```

Comment:
Would it not be better to just fix the config files where this is set with the ADAGUC_DB parameter?